packages/wrangler/src/dev/miniflare.ts

```diff
@@ -1052,6 +1059,18 @@ export async function buildMiniflareOptions(
 					entrypoint: name,
 					proxy: true,
 				})),
+				outboundService(request) {
+					if (request.headers.has("CF-Connecting-IP")) {
+						request.headers.delete("CF-Connecting-IP");
+					}
```

Comment:
This `outboundService` applies to the User Worker only. We were trying to add it to all workers in the `LocalRuntimeController` which might includes wrapped bindings and crash because it does not support `outboundService`.